src/puppetlabs/trapperkeeper/services/authorization/authorization_service.clj

```diff
@@ -0,0 +1,24 @@
+(ns puppetlabs.trapperkeeper.services.authorization.authorization-service
+  (:require [puppetlabs.trapperkeeper.core :refer [defservice]]
+            [puppetlabs.trapperkeeper.services :refer [service-context]]
+            [puppetlabs.trapperkeeper.authorization.ring-middleware :refer
+             [wrap-authorization-check]]
+            [puppetlabs.trapperkeeper.services.authorization.authorization-core :refer :all]))
+
+(defprotocol AuthorizationService
+  (wrap-handler [this handler]))
```

Comment:
`wrap-with-authorization-check` sounds like a better name to me as well.  I'd be good with that change.